config/routes.rb

```diff
@@ -6,5 +8,14 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+  root 'events#index'
+
+  get '/events', to: 'events#index', as: 'events'
+  post '/events', to: 'events#create'
+  get '/event/new', to: 'events#new', as: 'new_event'
+  get '/event/:id', to: 'events#show', as: 'event'
+  get '/event/:id/edit', to: 'events#edit', as: 'edit_event'
+  put '/event/:id', to: 'events#update'
+  patch '/event/:id', to: 'events#update'
+  delete '/event/:id', to: 'events#destroy'
```

Comment:
Checkout the rails source in routes to leave this more readable 
https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default